docusaurus.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  title: "Home Assistant Developer Documentation",
+  title: "Home Assistant Developer Docs",
   tagline: "All you need to start developing for Home Assistant",
```

Comment:
I think this might look better on the homepage

```suggestion
  tagline: "All you need to start developing",
```

Reply:
Current:

![image](https://user-images.githubusercontent.com/195327/94842220-6a5c6400-041b-11eb-9489-839b897742a3.png)

Suggestion:


![image](https://user-images.githubusercontent.com/195327/94842261-79431680-041b-11eb-90ca-ae76e82f3db3.png)

Reply:
Much better!